# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50  

# Position Controllers ---------------------------------------
wheel_right_joint_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: wheel_right_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
wheel_left_joint_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: wheel_left_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
prism_left_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: prism_left_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
prism_right_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: prism_right_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}